app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,13 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id]) 
+    @actors = @movie.actor_order_age
```

Comment:
Just FYI, this second instance var isn't strictly necessary as it's totally okay to call `@movie.actor_order_age` in the view itself! That's not an MVC violation. An MVC violation would be calling Active Record query methods in the view, but it's okay to call a model method name